devbox.go

```diff
@@ -85,6 +85,10 @@ func Open(dir string, writer io.Writer) (*Devbox, error) {
 	return box, nil
 }
 
+func (d *Devbox) GetConfigDir() string {
```

Comment:
In Go you usually leave out the Get prefix:

```suggestion
func (d *Devbox) ConfigDir() string {
```